src/node.cc

```diff
@@ -2364,7 +2366,8 @@ static void DLOpen(const FunctionCallbackInfo<Value>& args) {
   modlist_addon = mp;
 
   Local<String> exports_string = env->exports_string();
-  Local<Object> exports = module->Get(exports_string)->ToObject(env->isolate());
+  Local<Object> exports =
+      module->Get(exports_string)->ToObject(env->context()).ToLocalChecked();
```

Comment:
If you're already here, you can feel free to also change the `Get()` call to take a context argument :)

Reply:
> Get() call to take a context argument

I didn’t understand what you said.
Do you expect this change ?

```cpp
  Local<Object> exports = module->Get(exports_string).As<Object>();
```

Reply:
If you look at v8.h, you’ll see that there are two overloads of `->Get()`, one that takes a `context` argument and one that doesn’t (and it deprecated):

https://github.com/nodejs/node/blob/cbaf59c5b9a6063ee384b848aeb0aceb89082179/deps/v8/include/v8.h#L3166-L3168

At some point, we’re going to use the latter here instead of `module->Get(exports_string)`. Whether you want to do the conversion in this PR or not is up to you :)

If you want to see an example of how these conversions look like, you can take a look at https://github.com/nodejs/node/pull/16247.

Reply:
Thank you for detailed information.

> If you look at v8.h, 

I will check the header file properly after this.

I update to add `Local<Context>` argument.
Please check my changes !